### DISCUSSION TITLE
Daily Standup (Monday, April 10th)

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#81] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Performed [deploy 0 ](https://ibm-dashboard.vercel.app/)for our NextJS app

**What are your priorities for the next 24 hours?**:
 - Start working on the DB design with @DavBot02 @Int-Angel 

**Blockers**:
 - I am a bit busy with work

**Shoutouts**:
 - @Int-Angel because

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - Found some resources for learning NextJS

**What are your priorities for the next 24 hours?**:
 - Make progress on Mock

**Blockers**:
 - Lack of proper time management

**Shoutouts**:
 - @antonieto
